packages/suite-native-atoms/src/Input/Input.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { ReactNode, useRef, useState } from 'react';
+import { TouchableWithoutFeedback, TextInput } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Text } from '../Text';
+import { Box } from '../Box';
+
+type InputProps = {
+    value: string;
+    label: string;
+    onChange: (value: string) => void;
+    isDisabled?: boolean;
+    isInvalid?: boolean;
```

Comment:
let's agree to use `error` terminology in all things related to form field, so this prop shoud be named `isError` or `hasError`

Reply:
Ah, I really like isInvalid because it doesn't just restrict the usage to error (for example sometimes you want to indicate something wrong with the input without having validation check failing). 

But yeah, I will change it, just offering counter-argument against this name change.